src/style.css

```diff
@@ -0,0 +1,103 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(/fonts/Avenir-Book.ttf),
+    url(/fonts/Avenir-Heavy.ttf),
+    url(/fonts/Avenir.ttc);
+  font-weight: 900;
+}
+
+body {
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  padding: 20px 8px 8px;
+}
+
+.form__search--big {
+  padding: 23px 0;
+}
+
+.form__search--big:hover {
+  box-shadow: 0 4px 4px 0 rgba(0,0,0, 0.25);
+}
+
+.form__input--big::placeholder {
+  color: rgba(61, 78, 97, 1);
+  font-weight: 300;
+}
+
+.form__search--small {
+  padding: 10px 0;
+}
+
+.form__search--small:focus {
+  font-family: inherit;
+  font-weight: 900;
+}
+
+.form__search--small:hover {
+  box-shadow: 0 3px 8px 0 rgba(61,78,97, 0.2);
+}
+
+.form__input--small::placeholder {
+  color: rgba(61, 78, 97, 1);
+  font-weight: 300;
+}
+
+.form__search {
+  display: flex;
+  align-items: center;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  background: #fff;
+  box-shadow: 0 1px 8px 0 rgba(61,78,97, 0.1);
+  margin-bottom: 20px;
+  width: 100%;
+}
+
+.form__search:focus {
+  border-radius: 3px;
+  background: linear-gradient(180deg,#fff,#f6f6f7);
+}
+
+.form___icon {
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: inline-block;
+  width: 19px;
+  height: 19px;
+  margin-left: 26px;
+  margin-right: 17px;
+  flex-shrink: 0;
```

Comment:
Let's try to organize your CSS
I recommend you take a look at these styles code and use it in the feature
[9elements](https://9elements.com/css-rule-order) or [htmlAcademy](https://codeguide.academy/html-css.html#css-order)